site/book/01-getting-started/02-quickstart.md

```diff
@@ -86,14 +105,39 @@ You can preview the changes that will be made to the cluster:
 $ kpt live preview
 ```
 
-Finally, apply the resources to the cluster:
+Apply the resources to the cluster:
 
 ```shell
 $ kpt live apply --reconcile-timeout=15m
 ```
 
 This waits for the resources to be reconciled on the cluster by monitoring their status.
 
+## Update the package
+
+Subsequently, there might be updates to the remote `nginx` package which you want to rebase your local package against.
```

Comment:
I added a commit to rephrase some of this which I think got overwritten. Please apply that.

Reply:
Updated.